sdk/core/core-amqp/src/requestResponseLink.ts

```diff
@@ -176,7 +176,7 @@ export class RequestResponseLink implements ReqResLink {
             clearTimeout(waitTimer);
           }
           logger.verbose(
-            "[%s] request-messageId | '%s' == '%s' | response-correlationId.",
+            "[%s] request-messageId | '%s' === '%s' | response-correlationId.",
```

Comment:
Did eslint do this automatically? This eqeqeq is inside a string, so that would be strange.

Reply:
I probably did that myself. I will revert this specific one. Let me know if there's another!